### ISSUE TITLE
Main Activity never appear

### ISSUE THREAD

**Issue by snufflesrea** [Coffeegerm/BetterBarista#7]
Hello. I tried to deploy the app on my emulator. But it keep showing Splash Activity only. I have no idea what is happening.
----
After playing around with the code, I found that the DrinksFragment and TimerFragment doesnt load data properly

#### Expected behavior:
DrinksFragment shown as first view on Main Activity

#### Actual behavior:
DrinksFragment doesnt load data

#### How to reproduce:
- Change var coldLaunch on NavigationOnItemSelectedListener class to FALSE
- Deploy app on emulator
- Press more tab on bottom navigation
- Press drinks tab on bottom navigation

## LogCat
<pre><code>19-08-26 11:44:51.714 9094-9094/? I/feegerm.brew_i: Not late-enabling -Xcheck:jni (already on)
2019-08-26 11:44:51.874 9094-9094/? W/feegerm.brew_i: Unexpected CPU variant for X86 using defaults: x86
2019-08-26 11:44:52.018 9094-9094/io.github.coffeegerm.brew_it I/feegerm.brew_i: The ClassLoaderContext is a special shared library.
2019-08-26 11:44:52.657 9094-9094/io.github.coffeegerm.brew_it W/ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@8e0a96f
2019-08-26 11:44:52.725 9094-9094/io.github.coffeegerm.brew_it W/ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@86fe5bd
2019-08-26 11:44:52.829 9094-9094/io.github.coffeegerm.brew_it W/feegerm.brew_i: Accessing hidden method Landroid/view/View;->computeFitSystemWindows(Landroid/graphics/Rect;Landroid/graphics/Rect;)Z (light greylist, reflection)
2019-08-26 11:44:52.830 9094-9094/io.github.coffeegerm.brew_it W/feegerm.brew_i: Accessing hidden method Landroid/view/ViewGroup;->makeOptionalFitsSystemWindows()V (light greylist, reflection)
2019-08-26 11:44:52.879 9094-9094/io.github.coffeegerm.brew_it W/feegerm.brew_i: Accessing hidden method Landroid/graphics/FontFamily;-><init>()V (light greylist, reflection)
2019-08-26 11:44:52.879 9094-9094/io.github.coffeegerm.brew_it W/feegerm.brew_i: Accessing hidden method Landroid/graphics/FontFamily;->addFontFromAssetManager(Landroid/content/res/AssetManager;Ljava/lang/String;IZIII[Landroid/graphics/fonts/FontVariationAxis;)Z (light greylist, reflection)
2019-08-26 11:44:52.879 9094-9094/io.github.coffeegerm.brew_it W/feegerm.brew_i: Accessing hidden method Landroid/graphics/FontFamily;->addFontFromBuffer(Ljava/nio/ByteBuffer;I[Landroid/graphics/fonts/FontVariationAxis;II)Z (light greylist, reflection)
2019-08-26 11:44:52.879 9094-9094/io.github.coffeegerm.brew_it W/feegerm.brew_i: Accessing hidden method Landroid/graphics/FontFamily;->freeze()Z (light greylist, reflection)
2019-08-26 11:44:52.879 9094-9094/io.github.coffeegerm.brew_it W/feegerm.brew_i: Accessing hidden method Landroid/graphics/FontFamily;->abortCreation()V (light greylist, reflection)
2019-08-26 11:44:52.879 9094-9094/io.github.coffeegerm.brew_it W/feegerm.brew_i: Accessing hidden method Landroid/graphics/Typeface;->createFromFamiliesWithDefault([Landroid/graphics/FontFamily;II)Landroid/graphics/Typeface; (light greylist, reflection)</code></pre>

**Comment by Coffeegerm** [comment]
Hey, thanks for emailing me about this, the app hasn’t been worked on in some time as my job has required a lot of my time, but I have known about this issue for a while, if you would like to help in correcting it please let me know and we can look at it together to get the app up to speed with the latest API changes. Otherwise I can take a look at it this weekend and see whats going on.

**Comment by snufflesrea** [comment]
@Coffeegerm yeah. I am learning from this app. As I learn your code further, who knows I can solve the bug in the future.

**Comment by Coffeegerm** [comment]
Is there anything in particular that I can help with? The realm initialization was a quick way to get data into the application so I wouldn’t follow that for your own projects, not a very elegant way of doing things.
